daily-challenges/2024-10/15/khanh5923_15.spec.ts

```
@@ -0,0 +1,49 @@
+  }
+}
+test("daily/10/15_multiple-pages", async ({}) => {
+  const browser = await chromium.launch();
+  const context = await browser.newContext();
+  const pageTGDD = await context.newPage();
+  const pageDDV = await context.newPage();
+  // // TGDD
+  await pageTGDD.goto(`https://www.thegioididong.com/`);
+  await pageTGDD.locator(`//input[@id='skw']`).fill(`iPhone 16`);
+  await pageTGDD.click(`//button[@aria-label='button suggest search']`);
+  let priceTGDD = await pageTGDD
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'priceTGDD' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)